src/cli/tsconfig-for-init.json

```diff
@@ -1,22 +1,22 @@
 {
   "compilerOptions": {
+    // Enable latest features
     "lib": ["ESNext"],
     "target": "ESNext",
     "module": "ESNext",
     "moduleDetection": "force",
-    "jsx": "react-jsx",
-    "allowJs": true,
+    "jsx": "react-jsx", // support JSX
+    "allowJs": true, // allow importing `.js` from `.ts`
```

Comment:
i dont think this comment is fully accurate or helpful either.

Reply:
I agree with it. I just remained them as is from the previous tsconfig.